src/tests/end-to-end/binding_constraints/test_binding_constraints.cpp

```diff
@@ -185,7 +186,7 @@ class BCrhsConfig
 {
 public:
     BCrhsConfig() = delete;
-    BCrhsConfig(std::shared_ptr<BindingConstraint> BC, unsigned int nbTimeSeries);
```

Comment:
This helper class is clearly not an owner of the BC: we should provide it a reference instead of shared_ptr.
(possibly for later work since it has not been done in this PR)